doc/users/prev_whats_new/whats_new_1.5.rst

```diff
@@ -37,6 +37,7 @@ The upshot of this is that for interactive backends (including
 ``%matplotlib notebook``) in interactive mode (with ``plt.ion()``)
 
 .. ipython :: python
```

Comment:
I wonder if we should just have this as a code markup block, and not an ipython block? Otherwise we're just trying to keep code from MPL 1.5 running on new versions of MPL.

Reply:
That would be fine for me too, but that will removed the rendered plot.
You could also set to `:verbatim:` so it won't try to process.

Reply:
I'm not sure it's all that important to see the plot in _this_ specific case.

Reply:
Indeed, because you can not see it change!